README.md

```diff
@@ -15,6 +15,7 @@ This tool can be installed using [pip](https://pypi.org/project/pip/) from the P
 ```bash
 pip install cloudformation-cli cloudformation-cli-java-plugin cloudformation-cli-go-plugin cloudformation-cli-python-plugin
 ```
+You will need npm to run contract test (cfn test). Please ensure you have npm installed. Follow the instructions: https://www.npmjs.com/get-npm to get npm installed on your machine.
```

Comment:
not sure, if they can the way the code stands they will need npm because jsonata is getting installed before the transformation check is being made. It may be worth changing the code to make sure customers don't need to instal npm if they are not using the functionality.

Reply:
> the way the code stands they will need npm because jsonata is getting installed before the transformation check is being made

could we check and not install `jsonata` during `cfn test` if they're not using `propertyTransform`?